src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import movieData from './movieData';
+import Movies from './Movies'
+import './css/App.css'
+import NavBar from './NavBar';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: movieData
+    }
+    
+  }
+
+  render() {
+    return (
+      <main>
+        <NavBar />
+        <section className='movie-container'>
```

Comment:
We talked about this on Zoom, but your observation that we don't need the JSX `section` both here and in `Movies.js` is spot-on. After this is merged and I pull it down, I'll remove it here.